controller/user/manager.py

```diff
@@ -37,6 +38,28 @@ def get_user_roles() -> Dict[str, str]:
     return {str(u.id): u.role for u in user.get_all()}
 
 
+def get_admin_users(expand_mail_name: bool = False) -> List[User]:
```

Comment:
this two functions look similar only the call is different not sure if it makes sense to be one function with a parameter?